client/client-multi/client-c/src/kaa/kaa_logging.h

```diff
@@ -42,6 +42,54 @@ extern "C" {
     typedef struct kaa_log_collector        kaa_log_collector_t;
 #endif
 
+/**
+ * @brief Log bucket information structure.
+ * One or more log records aggregated into the bucket.
+ */
+typedef struct
+{
+    uint16_t bucket_id;     /**< ID of bucket present in storage. */
+    size_t   log_count;     /**< Logs left to upload across all buckets. */
```

Comment:
move log_count one line up. That doesn't change anything now, but has better chance to avoid padding in the future.